### PR TITLE
Fix #604: Add FDS data to ProcessEventRequest

### LINKED ISSUE #604: Add FDS data to ProcessEventRequest
[adapter-api.txt](https://github.com/wultra/enrollment-server/files/10389441/adapter-api.txt)

Follow up to #595

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/model/request/ProcessEventRequest.java [hunk 1]
```diff
@@ -49,7 +50,70 @@ public final class ProcessEventRequest {
...
+
+        /**
+         * Value to be obtained from Broadcom SDK on mobile device. Not filled yet.
+         */
+        private String deviceSignature;
+
+        @Override
+        public Map<String, Object> asMap() {
+            return Map.of(
+                    "locale", locale.getLanguage(),
```
Comment (banterCZ) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/model/request/ProcessEventRequest.java:107]: Here should be `language`.

File: enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingProcessEntityWrapper.java [hunk 2]
```diff
@@ -105,4 +123,15 @@ private void setValue(final String key, final Object value) {
             logger.warn("Problem to parse custom_data of process ID: {}", entity.getId(), e);
         }
     }
+
+    private String getValue(final String key) {
+        try {
+            logger.debug("Getting {} from custom_data: {} of process ID: {}", key, entity.getCustomData(), entity.getId());
+            final Map<String, Object> json = mapper.readValue(entity.getCustomData(), new TypeReference<>(){});
```
Comment (banterCZ) [inline enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingProcessEntityWrapper.java:130]: - [ ] Fix `Dereference null return value (NULL_RETURNS)`

File: enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingProcessEntityWrapper.java [hunk 3]
```diff
@@ -87,22 +88,60 @@ public void setIpAddress(final String ipAddress) {
...
+            logger.debug("Getting {} from custom_data: {} of process ID: {}", key, entity.getCustomData(), entity.getId());
+            final Map<String, Object> json = readValue();
+            return json.getOrDefault(key, "unknown").toString();
+        } catch (JsonProcessingException e) {
+            logger.warn("Problem to parse custom_data of process ID: {}", entity.getId(), e);
+            return "unknown";
+        }
+    }
+
+    private Map<String, Object> readValue() throws JsonProcessingException {
```
Comment (romanstrobl) [inline enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingProcessEntityWrapper.java:139]: Can we use a better method name? `readValue` is really too generic.

**Review by romanstrobl — APPROVED** [review]
Looks good, just one remark concerning method names.

**Pull request merged**